### PR TITLE
docs: explain how to override global automerge settings

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Explain how to clobber over a automerge config that gets `extended` from a global `.github/renovate.json` file

## Context:

As requested by @viceice in:

- https://github.com/renovatebot/renovate/discussions/11080#discussioncomment-1131362

Rough draft.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/automerge-configuration.md [hunk 1]
```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
 ### Codeowners
 
 Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.
+
+### Disabling automerge in select repos, rest of org should automerge
```
Comment (HonkingGoose) [inline docs/usage/automerge-configuration.md:142]: I'm not sure if this `h3` title is the correct summary of the section below, it's a bit long right now.

If you have any ideas, please suggest them!

File: docs/usage/automerge-configuration.md [hunk 2]
```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
...
+
+You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
+Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.
+To turn off automerge in full for select repositories, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:
```
Comment (HonkingGoose) [inline docs/usage/automerge-configuration.md:146]: I'm not sure if this properly explains the use-case for the config below, so please suggest changes to the wording.

**Review by HonkingGoose — COMMENTED** [review]
Help wanted to improve the text! 😉

**Review by viceice — DISMISSED** [review]

**Review by 74971030177 — COMMENTED** [review]
HonkingGoose:docs/11080-match-all-packageRules-to-override-others

File: docs/usage/automerge-configuration.md (hunk 1 above)
Comment (rarkins) [reply docs/usage/automerge-configuration.md:142]: Maybe `### Overriding global automerge`

Comment (HonkingGoose) [reply docs/usage/automerge-configuration.md:142]:
```suggestion
### Overriding global automerge
```

Let's go with that for now! 😄

File: docs/usage/automerge-configuration.md (hunk 2 above)
Comment (viceice) [reply docs/usage/automerge-configuration.md:146]:
```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.
To turn off automerge for all dependnecies of a selected repository, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:
```

Something like this?

Comment (HonkingGoose) [reply docs/usage/automerge-configuration.md]: I think your text is a bit clearer than mine, so I'll apply your suggestion. 😄

File: docs/usage/automerge-configuration.md [hunk 3]
```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
 ### Codeowners
 
 Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.
+
+### Overriding global automerge
+
+You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
```
Comment (viceice) [inline docs/usage/automerge-configuration.md:144]: Shouldn't we tell this shared preset? Not all people are using `.github` repo for this. 😏

Comment (HonkingGoose) [reply docs/usage/automerge-configuration.md:144]: I'm sorry, I really don't understand what you mean here. Can you try explaining again, but with different words?

Comment (rarkins) [reply docs/usage/automerge-configuration.md:144]: I think we've done enough describing already. There's nothing about this overriding which is specific to automerge anyway.

Comment (HonkingGoose) [reply docs/usage/automerge-configuration.md:144]:
```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
It does not matter where you've put the global config, the important point in this example is that you're extending from a global config that's somewhere else.
For this example we'll assume you put your config in a repository on GitHub, called `.github`.
```

How about this then? It explains a bit more, and tells readers that the important point is that you're extending from a config that's _somewhere else_.

**Review by viceice — APPROVED** [review]

**Pull request merged**